sources/platform/actors/publishing/monetize/pay_per_event.mdx

```diff
@@ -179,18 +171,46 @@ For new Actors, this event is added automatically as you can see on the followin
 
 #### Synthetic start event for existing Actors
 
-If you have existing Actors, you can add this event manually in Apify Console.
+If you have existing Actors, you can add this event manually in Apify Console in the Publication tab.
 
 #### Synthetic start event for Actors with start event
 
 Your Actor might already have a start event defined, such as `actor-start` or another variant of the event name. In this case, you can choose whether to use the synthetic start event or keep the existing start event.
 
-If you want to use the synthetic start event, remove the existing start event from your Actor and add the synthetic start event in Apify Console.
+If you want to use the synthetic start event, remove the existing start event from your Actor and add the synthetic start event in Apify Console in the Publication tab.
+
+### Set memory limits
+
+Set memory limits using `minMemoryMbytes` and `maxMemoryMbytes` in your [`actor.json`](https://docs.apify.com/platform/actors/development/actor-definition/actor-json) file to control platform usage costs.
+
+```json
+{
+    "actorSpecification": 1, 
+    "name": "name-of-my-scraper",
+    "version": "0.0",
+    "minMemoryMbytes": 512,
+    "maxMemoryMbytes": 1024,
```

Comment:
What's the best practice here @metalwarrior665 ? Should we recommend users to set rangers, or to rather set a single value and not let users tweak it, unless necessary? I'm wondering if the creators even realize that they can set the min and max to the same value.

Reply:
I think both approaches are valid. For now the safest is to have the same (low) value but it is nice to allow users to boost their run. Once we dynamic input-based memory, this will be mostly solved.

Reply:
I'm not leaning much in any direction. So I think keeping it like this is ok.